assets/js/modules/analytics/components/module/ModuleOverviewWidget/index.js

```diff
@@ -74,29 +76,46 @@ export default function ModuleOverviewWidget( { Widget, WidgetReportError } ) {
 	const statsReport = useSelect( ( select ) => select( MODULES_ANALYTICS ).getReport( statsArgs ) );
 	const statsError = useSelect( ( select ) => select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [ statsArgs ] ) );
 
-	if ( overviewError || statsError ) {
-		return <WidgetReportError statsError={ overviewError || statsError } />;
+	const isZero = isZeroReport( overviewReport );
+	const isError = overviewError || statsError;
+
+	if ( ! overviewLoaded || ! statsLoaded ) {
+		return <ProgressBar />;
 	}
 
+	const reportsAreValid = ! isError && ! isZero;
+
 	return (
-		<Widget Header={ Header } noPadding>
-			<Overview
-				loaded={ overviewLoaded }
-				report={ overviewReport }
-				selectedStat={ selectedStat }
-				handleStatSelection={ setSelectedState }
-			/>
+		<Widget
+			Header={ Header }
+			noPadding={ reportsAreValid }
```

Comment:
Maybe you can just keep this as
```suggestion
			noPadding={ ! isError && ! isZero }
```
? I can see why it makes sense to put it in a variable, but also this condition is super simple, and per my other comment IMO it's easier to understand when writing out these `isError` and `isZero` conditions always.

Reply:
Updated.